### PR TITLE
[WJ-929] Use alpine container for Postgres

### PR DESCRIPTION
**Pull request by emmiegit**
The PostgreSQL Alpine base image is lighter than the default (I believe Ubuntu) one, which will be faster and save money in deployment.

### CODE REVIEW

File: install/local/dev/postgres/Dockerfile [hunk 1]
```diff
@@ -7,22 +7,22 @@ ARG ENVIRONMENT="local"
...
 WORKDIR /src
 
 # Create system user
-RUN useradd wikijump --create-home --shell /bin/bash
+RUN adduser -S wikijump
 
 # Setup database
 USER postgres
-RUN pg_createcluster --user postgres --locale en_US.UTF-8 "${POSTGRES_VERSION}" "${POSTGRES_CLUSTER}"
+RUN initdb -D ${PGDATA} --locale=en_US.UTF-8
```
Comment (Monkatraz) [inline install/local/dev/postgres/Dockerfile]: Out of curiosity, why does a database have a locale? lol

**Review by Monkatraz — APPROVED** [review]

File: install/local/dev/postgres/Dockerfile (hunk 1 above)
Comment (emmiegit) [reply install/local/dev/postgres/Dockerfile]: Mostly for string encoding

**Comment by emmiegit** [comment]
Because of very bizarre build issues, I'm setting all our CI builds to use `DOCKER_BUILDKIT`.

Essentially, when running a build without it, the step after any filesystem changes were made in the database image were immediately reverted. You could `&& ls ${PGDATA}` to see that the directories had been created, but the step right after it with `RUN ls ${PGDATA}` showed no changes.

**Comment by emmiegit** [comment]
thanks @Monkatraz

**Pull request merged**